python_modules/libraries/dagster-dbt/dagster_dbt/dbt_resource.py

```diff
@@ -126,6 +126,18 @@ def seed(
         """
 
     @abstractmethod
+    def freshness(self, select: List[str] = None, **kwargs) -> DbtOutput:
```

Comment:
sorry I should have noticed this earlier, but this actually shouldn't be an abstract method of the DbtResource() class, because there are other DBT resources (namely the ones related to RPC) that do not implement this method. I think it's ok to have this be a CLI-only method for now, so just removing this should be ok.

Reply:
okay! can do